executables/darwin/.gitignore

```diff
@@ -0,0 +1,5 @@
+templates/grr-client*
```

Comment:
I think we can ignore everything here? replace with
`**`
same for other dirs below